boards/arm/mec1501modular_assy6885/mec1501modular_assy6885.dts

```diff
@@ -81,10 +81,30 @@
 
 &ps2_0 {
 	status = "okay";
+	pinctrl-0 = <&ps2_clk0b_gpio007 &ps2_dat0b_gpio010>;
+	pinctrl-names = "default";
+};
+
+&ps2_clk0b_gpio007 {
+	drive-open-drain;
+};
```

Comment:
as suggested in the pinctrl guide, customization of pre-defined nodes should happen in a `board-pinctrl.dtsi` file.

Reply:
added `board-pinctrl.dtsi` file